devices/FloatingBaseEstimatorDevice/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+# Copyright (C) 2020 Istituto Italiano di Tecnologia (IIT). All rights reserved.
+# This software may be modified and distributed under the terms of the
+# GNU Lesser General Public License v2.1 or any later version.
+
+if(FRAMEWORK_COMPILE_YarpImplementation)
+  cmake_minimum_required(VERSION 3.5)
```

Comment:
Is this required?

Reply:
This is removed in https://github.com/dic-iit/bipedal-locomotion-framework/pull/130/commits/4454ca878d36931fb1cf87acc79b4b95b5113462